python/bin/interpreter.bzl

```diff
@@ -0,0 +1,40 @@
+load("@bazel_skylib//lib:paths.bzl", "paths")
```

Comment:
Please move the implementation to `//python/private/interpreter.bzl` and re-export here